blueprints/hr_assistant/data/employee_data.json

```diff
@@ -0,0 +1,7226 @@
+        "subordinates": "Wilson K Adinolfi, Nader Barton, Beatrice Chace, Jean Crimmings, Miguel Estremera, Mildred Gentry, Joanne Handschiegl, George Johnson, Kramer Keatts, Marilyn Linares, Jennifer Medeiros, Clinton Owad, Quinn Rarrick, Nore Sadki, Timothy Sullivan, Anna Von Massenbach, Dianna Blount, Angela Erilus, Adrienne J Homberger, Susan Lundy, Maliki Moumanil, Constance Sloan"
+    },
+    {
+        "emp_name": "Bozzi, Charles",
+        "id": 1303054580,
+        "state": "MA",
+        "dob": "1970-03-10",
+        "age": 47,
+        "sex": "Male",
+        "maritaldesc": "Single",
+        "citizendesc": "US Citizen",
+        "racedesc": "Asian",
+        "doh": "2013-09-30",
+        "dot": "2014-08-07",
+        "rft": "retiring",
+        "employment_status": "Voluntarily Terminated",
+        "department": "Production       ",
+        "position": "Production Manager",
+        "money": 50.5,
+        "manager": "Janet King",
+        "employee_source": "Billboard",
+        "performance_score": "Fully Meets",
+        "first_name": "Charles",
+        "last_name": "Bozzi",
+        "subordinates": ""
+    },
+    {
+        "emp_name": "Butler, Webster  L",
+        "id": 1110029990,
+        "state": "MA",
+        "dob": "1983-08-09",
+        "age": 34,
+        "sex": "Male",
+        "maritaldesc": "Single",
+        "citizendesc": "US Citizen",
+        "racedesc": "White",
+        "doh": "2016-01-28",
+        "dot": "1800-01-01",
+        "rft": "N/A - still employed",
+        "employment_status": "Active",
+        "department": "Production       ",
+        "position": "Production Manager",
+        "money": 55.0,
+        "manager": "Janet King",
+        "employee_source": "Pay Per Click - Google",
+        "performance_score": "Exceeds",
+        "first_name": "Webster",
+        "last_name": "Butler",
+        "subordinates": ""
+    },
+    {
+        "emp_name": "Dunn, Amy  ",
+        "id": 1409070147,
+        "state": "MA",
+        "dob": "1973-11-28",
+        "age": 44,
+        "sex": "Female",
+        "maritaldesc": "Single",
+        "citizendesc": "US Citizen",
+        "racedesc": "White",
+        "doh": "2014-09-18",
+        "dot": "1800-01-01",
+        "rft": "N/A - still employed",
+        "employment_status": "Active",
+        "department": "Production       ",
+        "position": "Production Manager",
+        "money": 51.0,
+        "manager": "Janet King",
+        "employee_source": "Search Engine - Google Bing Yahoo",
+        "performance_score": "Fully Meets",
+        "first_name": "Amy",
+        "last_name": "Dunn",
+        "subordinates": "Linda Anderson, Sean Bernstein, Enola Chivukula, Carl Desimone, Nilson Fernandes, Evelyn Girifalco, Kara Harrison, Alexandra Kirill, Samuel MacLennan, Horia Ndzi, Shakira Perry, Alain Robinson, Seffi Shields, Sophia Theamstern, Barry Wilber, Joelle Burke, Michael J Fitzpatrick, Ming Huynh, Karen Mancuso, Ermine Pelletier, Cybil Trzeciak"
```

Comment:
Cleaner to have this as a structured list of strings rather than a single string with comma separated names